test/graph_rewrites/config_for_grappler_test.cc

```diff
@@ -70,13 +70,9 @@ TEST(GrapplerConfig, RConfig1) {
   graph.ToGraphDef(&item.graph);
   ConfigProto config_proto;
   auto backend_name = AttrValue();
-  backend_name.set_s("NNPI");
-  auto device_id = AttrValue();
-  device_id.set_s("1");
-  auto num_ice_cores = AttrValue();
-  num_ice_cores.set_s("4");
-  auto max_batch_size = AttrValue();
-  max_batch_size.set_s("64");
+  backend_name.set_s("CPU");
```

Comment:
Had to redo the test to be for CPU backend since a check was added to see if the requested backend is supported and the earlier test would fail.